src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -648,7 +652,7 @@ private void handleInvalidFileType(Handler<AsyncResult<Response>> asyncResultHan
 
   private void handleInvalidProfilePictureSize(Handler<AsyncResult<Response>> asyncResultHandler) {
     asyncResultHandler.handle(
-      succeededFuture(PostUsersProfilePictureResponse.respond500WithApplicationJson("Requested file size should be within allowed size 0.1-10.0 megabytes")));
+      succeededFuture(PostUsersProfilePictureResponse.respond500WithApplicationJson("Requested file size should be within allowed size")));
```

Comment:
Can we print the max file size in the response. It will be very clear the max file size allowed to the end user.

Reply:
fixed